node/service/src/grandpa_support.rs

```diff
@@ -279,20 +278,12 @@ mod tests {
 
 		// add 10 blocks
 		push_blocks(10);
-		assert_eq!(
-			client.info().best_number,
-			10,
-		);
+		assert_eq!(client.info().best_number, 10,);
```

Comment:
was this done intentionally or is an automatic formatter being used?

Reply:
In this case rustfmt generated the change.